net/modemmanager/files/modemmanager.common

```diff
@@ -235,24 +235,20 @@ mm_report_modem_wait() {
 # Cleanup interfaces
 
 mm_cleanup_interfaces() {
-	local modemlist modemlength idx modeminfo modemsysfspath
-
-	modemlist=$(mmcli --list-modems --output-keyvalue)
-	[ -n "${modemlist}" ] || return 0
-
-	modemlength=$(modemmanager_get_field "${modemlist}" "modem-list.length")
-
-	# do nothing if no modem reported
-	[ -n "${modemlength}" ] && [ "${modemlength}" -ge 1 ] && {
-		idx=1
-		while [ $idx -le "$modemlength" ]; do
-			modempath=$(modemmanager_get_field "${modemlist}" "modem-list.value\[$idx\]")
-			modeminfo=$(mmcli --modem "${modempath}" --output-keyvalue)
-			modemsysfspath=$(modemmanager_get_field "${modeminfo}" "modem.generic.device")
-			mm_cleanup_interface_by_sysfspath "${modemsysfspath}"
-			idx=$((idx + 1))
-		done
-	}
+	local sysfs_path status
+
+	# Do nothing if there is no sysfs cache
+	[ -f "${MODEMMANAGER_SYSFS_CACHE}" ] || return
+
+	while IFS= read -r sysfs_cache_line; do
+		sysfs_path=$(echo "${sysfs_cache_line}" | awk '{print $1}')
+		status=$(echo "${sysfs_cache_line}" | awk '{print $2}')
+
+		if [ "${status}" = "processed" ]; then
+			mm_log "info" "call cleanup for: ${sysfs_path}"
```

Comment:
I don't think this should be "info" level

Reply:
Yes this should be a "debug" level